src/platform/packages/shared/kbn-unified-field-list/src/components/field_item_button/field_item_button.tsx

```diff
@@ -260,3 +260,5 @@ function FieldConflictInfoIcon({
     </EuiToolTip>
   );
 }
+
+export type GenericFieldItemButtonType = typeof FieldItemButton;
```

Comment:
I'm curious why we need this type, is it not typed correctly in the dynamic import below? Maybe it would work to do a default export from here instead to avoid deconstructing the import and casting.